### PR TITLE
Navbar for all pages

### PR DESCRIPTION
**Pull request by drenetic**
# NAVIGATION BAR ADDED FOR ALL PAGES

## Description

* Navigation bar added for all pages as a template.
* Updated drop-down items (Edit Profile, Log Out)

## Has this been tested? How?

* Tested w/ 'rails s' + yarn build --watch to compile webpack application.js (otherwise dropdown functionality won't work) 

## Types of change

- [X] New feature (change which adds functionality)

- [ ] Bug fix (change which fixes an issue)

- [ ] Improvement/refactoring (change that doesn't add any features but makes things better)

- [X] Styling (style changes that don't affect functionality)

## Checklist before submitting

- [X] Satisfies card on Trello

- [ ] Has had a another team mate review your design.

- [X] No known defects

- [x] Appropriate tests completed (e.g. core functionality, edge cases)

- [X] Code has been formatted with nicely and contains proper indentation.

- [X] Code comments as appropriate

- [X] `master` branch has been merged into the feature branch and tested (if any newer commits on `master`)

### CODE REVIEW

**Comment by drenetic** [comment]
[A signed in user should be able to edit their account details and delete their account if needed.](https://trello.com/c/MFM69D9h/18-a-signed-in-user-should-be-able-to-edit-their-account-details-and-delete-their-account-if-needed)

**Comment by drenetic** [comment]
[Create a basic Navbar to be used in all pages](https://trello.com/c/VrTEHNN5/17-create-a-basic-navbar-to-be-used-in-all-pages)

**Comment by drenetic** [comment]
[Create a basic Navbar to be used in all pages](https://trello.com/c/VrTEHNN5/17-create-a-basic-navbar-to-be-used-in-all-pages)

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <div class="container-fluid">
+    <%= link_to "#", class: "navbar-brand" do %>
+      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
```
[the remark this answers is not in the document]
Comment (drenetic) [reply app/views/shared/_navbar.html.erb:4]: this will be done when we get to styling.

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto">
+        <% if user_signed_in? %>
+          <li class="nav-item active">
+            <%= link_to "Home", "#", class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "Messages", "#", class: "nav-link" %>
```
Comment (learner1unknown) [inline app/views/shared/_navbar.html.erb:18]: Change messages to Bookings

**Review by learner1unknown — COMMENTED** [review]
Good job see comments

**Review by learner1unknown — APPROVED** [review]
